geode-book/config.yml

```diff
@@ -28,8 +28,9 @@ template_variables:
   product_name_long: Apache Geode
   product_name: Geode
   product_name_lowercase: geode
-  product_version: '1.15.0'
+  product_version: '1.15'
   product_version_nodot: '115'
+  product_version_old_minor: '1.14'
```

Comment:
If this variable is intended to be the previous minor version of the current product version, I think it will be more intuitive call it `product_vesion_previous_minor`.

Reply:
It doesn't have to be the immediately previous minor, any earlier version will do. I thought it best from a maintenance standpoint to take the more flexible approach.